scripts/activate.sh

```diff
@@ -40,6 +40,6 @@ for env in $envs; do
     npm dist-tag add $module@$version "$tag-$env" --otp="$twofactorcode";
 done;
 
-sleep 5;
+$DIR/grabthar-verify-npm-publish "active-production"
```

Comment:
Wanna loop over `$envs` right? it's not mandatory to activate to production

Reply:
Can we also pass in $version since the version being activated might not match the version in package.json.

Reply:
Thanks for the feedback! Yes, on it now.